sros2/test/sros2/commands/security/verbs/test_generate_policy.py

```diff
@@ -24,20 +24,21 @@
 from test_msgs.srv import Empty
 
 
-# TODO(ivanpauno): reactivate this test after updating generate policy
-@pytest.mark.skip(reason='temporarily deactivated')
 def test_generate_policy_topics():
     with tempfile.TemporaryDirectory() as tmpdir:
+        TEST_ENCLAVE = '/foo/bar'
+        TEST_NODE_NAMESPACE = '/'
+        TEST_NAME = 'test_generate_policy_topics'
+        TEST_NODE_NAME = TEST_NAME + '_node'
```

Comment:
nit: these are formatted like globals, they should probably be lower-case.

Reply:
:+1: they started as globals and didnt change case when they became locals.. updated in https://github.com/ros2/sros2/pull/203/commits/cab6123d7dafe8a9d16e48266c229bb884960346